### DISCUSSION TITLE
2023-08-30 Weekly Meeting 122 Notes/Agenda

### DISCUSSION THREAD

**Discussion by indiebio** [omigroup/omigroup#414] [Weekly Meetings]
Weekly Meeting 122 is happening on Wednesday, August 30, 2023 from 9:00 AM PDT (16:00 UTC) - 10:00 AM PDT (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat.

Who we are and why we're here:
The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal.

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

Agenda

1. Past meetings transcribed and fireflie'd, SO GOOD! thanks jin. Need this link pinned somewhere, even on website mainpage?
https://hackmd.io/@XR/omigroup/https%3A%2F%2Fhackmd.io%2F%40XR%2Fomi_8-23-23
Working groups update: 
2. glTF
3. MSF
4. <agenda in progress :) >
5. M3 open day recap: https://mirror.xyz/m3org.eth/mTtZNs7XXNAQfLeqn6VCc7Nj5-9PIDBjlwNZ88EBb6U
metaverse festival:
7. NAME? 
      1. "Metaverse festival"
      2. OMIFES
      3. . OMIFEST / OmiFest: Open Metaverse Interoperability Festival
      4. MetaVerseCon: A conference dedicated to building an open and interoperable metaverse.
      5. OpenMeta Expo: An exposition showcasing the efforts towards an open metaverse.
      6. InteropFest: Celebrating the strides in interoperability in the metaverse.
      7. VersaThon: A hackathon-style event focused on creating an open metaverse.
3. Platform?
   1. concerns with hyperfy
   2. but portals possible. Turning it around, hyperfy could be a portal, with different starting platform.
   3.  an installer pack with desired platform? e.g. Like we ship overte, v-sekai and a web browser (could be a web-view) 
4. Related: 
Social VR 2023 survey: https://docs.google.com/forms/d/e/1FAIpQLSfzkJNb1D085UZ6Nwbnq8RVf_jqQvQ4UuzozaBGiF9gp0ukPg/viewform

**Comment by madjin** [comment]
I got drag n drop pallete ready for burning man themed buildathon, 1pm EST friday is when planning to do a collaborative build event with M3 and Neon Buidl
![Screenshot from 2023-08-30 11-26-17](https://github.com/omigroup/omigroup/assets/32600939/cea750a4-7eb5-4d7e-bb7c-0bfb9dafd3c9)

**Comment by indiebio** [comment]
1. meeting archive awesome! summaries more than raw markdown, and also useful to train AI models for future problemsolving etc.

**Reply by indiebio** [reply]
Also note that those notes supplement strongly these on-the-fly meetings notes :)

**Comment by indiebio** [comment]
2. Need more people to show up for glTF. 
WIFM? Aaron: frustrated by content sucked into Unity packages. Want more openness. 
so what is the strategy ?
Started looking at vehicles, but physics need to be done first.
How do we get people excited about physics then?
gtTF is ratified, but the physics spec the team is working on is not ratified yet.

TODO: Async the next month, send links to  Hyperfy / Spatial etc the github links about the glTF Physics issues.

TODO: Brainstorm if we can get e.g. physics academics interested to collaborate. They are niche and may be OK with new standards.

TODO: Summary page (on hackmd) of these github issue links, tell the story. 

ideas: show integrations between our glTF Physics and other standards, to show that it works well together.

dovetail the more technical hackathon proof of concept models, and the more community focused events. To combine with e.g. speakers, hackathon, and show-and-tell. - jimmy6dof to write this idea up in a bit to unpack the thinking.

**Comment by indiebio** [comment]
5. We've been doing virtual events for 5 years now, we're pro at it now. We're looking to expand, help and run events with other communities. Can do on any platform, is being recorded and documented and summarised. Yay!

**Comment by indiebio** [comment]
6. Metaverse name: There is a poll open for the name, open until Friday 1 September. 
https://discord.com/channels/770382203782692945/976535598313644113/1146487699486801951

The poll will also be shared / duplicated / cross posted on Twitter etc. for input (results may be weighted).

**Comment by jimmy6DOF** [comment]
MONHTLY EVENT PROGRAM PLAN
Regular mini-Events as build up to .... "2024 Open Ready Player Playa" maybe a countdown ?

THEMES
- Guest Speaker(s) (Headliners)
- Workshops, Show & Tells, & Demo Day
One can be the Aftershow/Preshow sidecar event for the other depending on content weights
Can promote them together & can dovetail the themes 

Synergy with like minded Work Groups :  Attendance
- Virtual Worlds Society / Spatial Ape / XR Guild / WebXR Discord / BabalyonJS Forum / Godot Discord / MetaTraversal /  M3 / WebXR Awards / Reality Innovators Network / AWE  
- OM3 / OMF / MSF 

Synergy with Metaverse Providers / Platforms :  Venues
- RP1 / M3 / (VRChat) 

Output to vTube/Twitch/Podcast facilities (invite creators)

Guest Invitation Show Runners chase down and sell the slot to speakers
Invitations & Attendance : Channel collateral through partners & beat the street
Publish Calendar & Update
